x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/external/action/huggingface/HuggingFaceActionCreatorTests.java

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-package org.elasticsearch.xpack.inference.external.huggingface;
+package org.elasticsearch.xpack.inference.external.action.huggingface;
```

Comment:
Github is kind of confused here, I didn't move this file. I removed the `HuggingFaceClient` and then tests and moved a few of the tests into `HuggingFaceActionCreatorTests`.